src/common/components/profile-preview/index.scss

```diff
@@ -14,8 +12,32 @@
     border-top-right-radius: 0px !important;
   }
 
-  @media (min-width: 577px) and (max-width: 740px) {
-    width: 45%;
+  @media (min-width: 577px) and (max-width: 1000px) {
+    width: 100%;
+  }
+}
+
+.popper-container {
+  width: 30% !important;
+  z-index: 2 !important;
+  padding: 30px 0px;
+  @media (max-width: $sm-break) {
+    width: 100% !important;
+    background-color: rgba(0, 0, 0, 0.1);
```

Comment:
Same comment for this color as above.

Reply:
@ghazanfar-BrilliantStar 
I have created a profile popover component which has a profile preview component as a child. This way we have small components and we can manage them easily. Please review the PR and let me know if needs any more improvement.

Reply:
@ghazanfar-BrilliantStar 
I have seen some other colors are mentioned in a file as I have mentioned here. If you want to add colors to the theme then please give me some idea that how you guys are managing the theme so I can follow along.